input/fsh/Profiles clinical datapoints/TreatmentVariables.fsh

```diff
@@ -0,0 +1,449 @@
+* reasonReference only Reference (PrimaryBreastCancerCondition)
+* medication[x] and subject and effectivePeriod and reasonCode and reasonReference MS
+
+Instance: ChemotherapyPatient147
+InstanceOf: Chemotherapy
+Description: "Example of chemotherapy of the breast cancer patient received during the last year"
+* status = MedicationAdministrationStatusCS#completed
+* medicationCodeableConcept = NullFlavor#OTH "other"
+* subject = Reference(BreastCancerPatient147)
+* reasonCode = SCT#373846009 "Adjuvant - intent"
+* reasonReference = Reference(PrimaryBreastCancerPatient147)
+* effectivePeriod.start = "2019-01-09"
+* effectivePeriod.end = "2019-09-04"
+
+Mapping: ChemotherapyToICHOM
+Source:	Chemotherapy
+Target: "https://connect.ichom.org/patient-centered-outcome-measures/breast-cancer"
+Id: chemotherapy-mapping
+Title: "Chemotherapy to ICHOM set"
+Description: "Mapping of chemotherapy to the ICHOM breast cancer PCOM set" 	
+* -> "Treatment"
+* effectivePeriod.start -> "Chemotherapy start date" 
+* effectivePeriod.end -> "Chemotherapy stop date" 
+* medication[x] -> "Type of chemotherapy"
+* reasonCode -> "Chemotherapy"
+
+Profile: ChemotherapyRequest
+Parent: MedicationRequest
+Id: chemotherapy-medication-request
+Title: "Chemotherapy request"
+Description: "Represents the order or request for chemotherapy of a breast cancer patient. This profile is in alignment with mCODE."
+* medication[x] only CodeableConcept 
+* medication[x] from ChemoTherapyTypeVS (preferred)
+* subject only Reference(BreastCancerPatient)
+* reasonCode from TherapyIntentVS (required)
+* reasonReference only Reference (PrimaryBreastCancerCondition)
+* medication[x] and subject and dispenseRequest.validityPeriod and reasonCode and reasonReference MS
+
+Instance: ChemotherapyRequestPatient147
+InstanceOf: ChemotherapyRequest
+Description: "Example of chemotherapy request of the breast cancer patient received during the last year"
+* status = MedicationRequestStatusCS#completed
+* intent = MedicationRequestIntentCS#order
+* medicationCodeableConcept = NullFlavor#OTH "other"
+* subject = Reference(BreastCancerPatient147)
+* reasonCode = SCT#373846009 "Adjuvant - intent"
+* reasonReference = Reference(PrimaryBreastCancerPatient147)
+* dispenseRequest.validityPeriod.start = "2019-01-09"
+* dispenseRequest.validityPeriod.end = "2019-09-04"
+
+Mapping: ChemotherapyRequestToICHOM
+Source:	ChemotherapyRequest
+Target: "https://connect.ichom.org/patient-centered-outcome-measures/breast-cancer"
+Id: chemotherapy-request-mapping
+Title: "Chemotherapy request to ICHOM set"
+Description: "Mapping of chemotherapy request to the ICHOM breast cancer PCOM set" 	
+* -> "Treatment"
+* dispenseRequest.validityPeriod.start -> "Chemotherapy start date" 
+* dispenseRequest.validityPeriod.end -> "Chemotherapy stop date" 
+* medication[x] -> "Type of chemotherapy"
+* reasonCode -> "Chemotherapy"
+
+// HORMONAL THERAPY
+Profile: Hormonaltherapy
+Parent: MedicationAdministration
```

Comment:
For Radiotherapy there is a Procedure profile. Why here only MedicationAdministration? Shouldn't there also be a Procedure linked to MedicationAdministration?